server/internal/registry/server.go

```diff
@@ -341,7 +346,9 @@ func (s *Local) handlePull(w http.ResponseWriter, r *http.Request) error {
 					status = fmt.Sprintf("error: %v", err)
 				}
 				enc.Encode(progressUpdateJSON{Status: status})
+				return nil
 			}
+			enc.Encode(progressUpdateJSON{Status: "Done."})
```

Comment:
Could we remove sending this status line? I think in almost every case (already downloaded, needing to download the model) the progress bar is perfect

Reply:
This was request by @mchiang0610. He has reverted his thinking. I've removed it.